server.go

```diff
@@ -136,33 +144,21 @@ func NewFromConfig(conf Config) (ret Server, err error) {
 			logrus.PanicLevel,
 		},
 	})
-	log.WithField("version", VERSION).Info("Starting server")
 
-	log.WithField("number", conf.NumWorkers).Info("Starting workers")
+	log.WithField("number", conf.NumWorkers).Info("Preparing workers")
+	// Allocate the slice, we'll fill it with workers later.
 	ret.Workers = make([]*Worker, conf.NumWorkers)
```

Comment:
Why not allocate this directly before it is used in `Start()`? Seems like it would be a bit simpler and maybe less error prone.